analyses/cell-type-wilms-tumor-14/scripts/utils/01_anchor_transfer_seurat_functions.R

```diff
@@ -40,7 +45,11 @@ run_anchorTrans <- function(path_anal, scratch_out_dir, results_out_dir, plots_o
       mutate(annot = compartment)
   } else {
     ref_obj@meta.data <- ref_obj@meta.data %>%
-      mutate(annot = celltype)
+      mutate(annot = celltype) %>%
+      mutate(annot = case_when(compartment == "stroma" ~ "stroma",
+                              compartment == "immune" ~ "immune",
+                              compartment == "endothelium" ~ "endothelium",
+                              TRUE ~ annot))
```

Comment:
I'm not sure I understand the logic here. While I can understand that at the end you might not want to retain the fine annotations, I feel like you would want to keep them during the anchor transfer step, and maybe combine the two labels afterward?

Reply:
Thanks for the comment.

My goal here is to improve the prediction scores for `stroma`, `immune`, and `endothelium` and make the annotation cleaner in visualization. If detailed cell types are included here, we may end up getting no cell types related to `stroma`, `immune`, or `endothelium` other than `Unknown`, for two reasons: (i) In general too few `stroma`, `immune`, and `endothelium` cells are present in the dataset; (ii) Prediction scores for detailed cell types are getting lower which creates a lot of `Unknown`.

However, I'm not sure if this is the correct thing to do. As you mentioned, I could also try to keep cell types during anchor transfer step, and combine labels later on (or even keep all `Unknown`s as is?). @jashapiro Any suggestions?